syntax = "proto3";

message RequestMsg
{
    int32 cmdType = 1;
    bytes clientID = 2;
    bytes serverID = 3;
    bytes sign = 4;
    bytes data = 5;
}

message RespondMsg
{
    bool status = 1; // 返回值
    int32 seckeyID = 2;  		// 对称密钥编号 keysn
    bytes clientID = 3; 	// 客户端编号
    bytes serverID = 4; 	// 服务器编号
    bytes data = 5; 	// 服务器端随机数
}
